installer/functions/common.ts

```diff
@@ -0,0 +1,58 @@
+import { Context, TwilioClient } from "@twilio-labs/serverless-runtime-types/types";
+import { ApplicationContext } from "../src/constants/interface";
+
+// constants
+export const IS_LOCALHOST = "IS_LOCALHOST";
+export const ACCOUNT_SID = "ACCOUNT_SID";
+export const AUTH_TOKEN = "AUTH_TOKEN";
+export const LOCAL_HOST = "localhost:";
+
+export function getParam(context: Context, key: string) {
+  const client = context.getTwilioClient();
+  const appContext = context as ApplicationContext;
+  try {
+    switch (key) {
+      case IS_LOCALHOST: {
+        return isLocalhost(appContext);
+      }
+      case ACCOUNT_SID: {
+        return getAccountSid(appContext, client);
+      }
+      case AUTH_TOKEN: {
+        return getAuthToken(appContext, client);
+      }
+    }
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+function isLocalhost(context: ApplicationContext): string {
+  return context.DOMAIN_NAME.startsWith('localhost:') ? 'yes' : 'no';
+}
+
+async function getAccountSid(context: ApplicationContext, client: TwilioClient): Promise<string> {
+  try {
+    if (context.ACCOUNT_SID) return context.ACCOUNT_SID;
+    const account = await client.api.accounts
+      .list({status: 'active', limit: 20})
+      .then(accounts => accounts[0]);
+    return account.sid;
+  } catch (err) {
+    console.error(err);
+    throw(err);
```

Comment:
should just be `throw err;`? but also i'd assume that the calling function of `getAccountSid` should be the one `console.error`ing, this catch essentially does nothing